### ISSUE TITLE
Não é possivel inserir um endereço

### ISSUE THREAD

**Issue by decko** [hackultura/login-cidadao#4]
No menu principal, há a opção "Endereço". Clicando nela, é aberta uma nova pagina com o botão "Adicionar Endereço". Ao clicar nesse botão, é retornada uma pagina de erro padrão.

**Comment by gilsondev** [comment]
Ele descreve esse erro?

**Comment by decko** [comment]
Nope. Mostra apenas uma pagina padrão de erro. Talvez apareça algo nos logs.

**Comment by gilsondev** [comment]
Vou dar uma olhada.
